apps/elasticsearch/deployers/es-statefulset.yaml

```diff
@@ -0,0 +1,117 @@
+# RBAC authn and authz
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: elasticsearch-logging
+  # namespace: logging
+  labels:
+    lkey: lvalue
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: elasticsearch-logging
+  labels:
+    lkey: lvalue
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - "*"
+  verbs:
+  - "get"
+  - "list"
+  - "watch"
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+        #  namespace: logging
+  name: elasticsearch-logging
+  labels:
+    lkey: lvalue
+subjects:
+- kind: ServiceAccount
+  name: elasticsearch-logging
+  namespace: logging
```

Comment:
Why different namespace is being used here?

Reply:
@gprasath i will remove this namespaces becouse we are already deploying it on certain namespaces.....looks like different services are also having seperate namespaces..